Makefile

```diff
@@ -0,0 +1,12 @@
+OCAML_SRCFILES = $(shell git ls-files "*.ml" "*.mli")
+REASON_SRCFILES = $(shell git ls-files "*.re" "*.rei")
+
+build:
+	yarn build
+.PHONY: build
+
+fmt:
+	esy ocamlformat --inplace --enable-outside-detected-project $(OCAML_SRCFILES)
+	esy refmt --in-place $(REASON_SRCFILES)
```

Comment:
Actually, I don't use esy so this is making formatting unusable for me. If someone wants to use ocamlformat from esy, they should just run `$ esy make fmt`.

Reply:
Updated.